ui/pages/routes/routes.component.js

```diff
@@ -371,11 +370,7 @@ export default class Routes extends Component {
           path={`${CONFIRM_TRANSACTION_ROUTE}/:id?`}
           component={ConfirmTransaction}
         />
-        <Authenticated
-          path={SEND_ROUTE}
-          component={process.env.MULTICHAIN ? SendPage : SendTransactionScreen}
-          exact
-        />
+        <Authenticated path={SEND_ROUTE} component={SendPage} exact />
```

Comment:
With this it seems that we are no longer using the SendTransactionScreen page anywhere. Are we going to remove all the dead code that will linger around after this PR?